docs/subnets/create-a-fuji-subnet.md

```diff
@@ -51,219 +56,613 @@ The response should look something like:
+These addresses are known as your control keys. You will also
+set how many control keys are required to add a validator.
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Set control keys:
+  ▸ Add control key
+    Done
+    Cancel
 ```
 
-By default, the private key in `.subnet-cli.pk` file which was created from [the step above](#subnet-cli-private-key) is used to pay the cost of this transaction.
-You can use `--private-key-path` to specify a different file.
+Enter at `Add control key` and provide at least one key. You can enter multiple addresses, we'll use just one here. When no more addresses need to be added, hit `Done`.
+(The address provided here is intentionally invalid. The address has a checksum and the tool will make sure it's a valid address).
+
+```bash
+✔ Add control key
+Enter P-Chain address (Ex: `P-...`): P-fuji1vaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaasz
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Set control keys:
+    Add control key
+  ▸ Done
+    Cancel
+```
+
+Finally, we need to define the threshold of how many keys are required for a change to be valid (there is some input validation). For example, if 1 key only is needed, as above, we'll enter just 1. The threshold can be arbitrary depending on the needs, e.g. 2 of 4 addresses, 1 of 3, 3 of 5, etc.
+
+```bash
+✔ Enter required number of control key signatures to add a validator: 1
+```
+
+Here the wizard completes, and the transaction is attempted.
+
+If the private key is not funded or does not have enough funds, we'll get an error message:
+
+```bash
+Error: insufficient funds: provided UTXOs need 100000000 more units of asset "U8iRqJoiJm8xZHAacmvYyZVwqQx6uDNtQeP3CQ6fcgQk3JqnK"
+```
+
+If the private key is funded, but the **control key** is incorrect (not controlled by the private key), the subnet will be created, but _not the blockchain_:
+
+```bash
+Subnet has been created with ID: 2EkPnvnDiLgudnf8NjtxaNcVFtdAAnUPvaoNBrc9WG5tNmmfaK. Now creating blockchain...
+Error: insufficient authorization
+```
+
+Therefore we need to provide a control key which we have indeed control of, and then it succeeds:
+
+```bash
+Subnet has been created with ID: 2b175hLJhGdj3CzgXENso9CmwMgejaCQXhMFzBsm8hXbH2MF7H. Now creating blockchain...
+Endpoint for blockchain "2XDnKyAEr1RhhWpTpMXqrjeejN23vETmDykVzkb4PrU1fQjewh" with VM ID "tGBrMADESojmu5Et9CpbGCrmVf9fiAJtZM5ZJ3YVDj5JTu2qw": https://api.avax-test.network/ext/bc/2XDnKyAEr1RhhWpTpMXqrjeejN23vETmDykVzkb4PrU1fQjewh/rpc
+```
+
+Well done! You have just created your own subnet with your own Subnet EVM running on `fuji`!
+
+## Add a validator
 
-As mentioned before, the `vm-id` was generated by calling `subnet-cli create VMID subnetevm`. You can
-use any value here, the only important thing is to make sure the binary you have generated has the same name.
+This new subnet is cool - but it doesn't have any dedicated validators yet! Let's add one by running the `addValidator` command and adding the name of our subnet.
```

Comment:
I think we should be a little more explicit that this command only whitelists the validator, it does not run it.

Reply:
Changed.